spec/controllers/account/admin/osbbs_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe Account::Admin::OsbbsController, type: :controller do
+  let!(:user) { create(:user) }
+  let!(:osbb) { create(:osbb) }
+  let!(:valid_params) { attributes_for :osbb }
+  let!(:invalid_params) { { name: '' } }
+
+  before { sign_in user }
```

Comment:
maybe, create spec helper method that create and sing in user

Reply:
I create controller_macros with login_user & login_admin (for the future), but this methods  I must put in every describe section, and I think with let!(:user) {create :user} and before { sign_in user } there was less rows of code. So I don't know if I need to push these changes ...